src/screencast/input.ts

```diff
@@ -52,6 +52,13 @@ export class ScreencastInputHandler {
             keyboardEvent.preventDefault();
             keyboardEvent.stopPropagation();
         }
+        if (keyboardEvent.ctrlKey && (keyboardEvent.key === 'c' || keyboardEvent.key === 'x') && keyboardEvent.type === 'keydown') {
+            this.cdpConnection.sendMessageToBackend('Runtime.evaluate', {
+                expression: 'document.getSelection().toString()',
+            }, (text: {result: {value: string}}): string => {
+                return text.result.value;
```

Comment:
Does this `return` do anything? It looks like your changes to `cdp.ts` read the value directly out of the message and don't look at the return value from the callback.